src/scheduler/src/main/resources/application.yml

```diff
@@ -118,15 +118,27 @@ scheduler:
         port: 8081
         unprotectedEndpoints: /health/prometheus,/health/liveness
         username: "test-user"
+
 ############################################################
 #    Server SSL keys
 ############################################################
 
 server:
+    forward-headers-strategy: framework
```

Comment:
I would suggest here : native so that it support the webserver feature 
Also, the recommendation is to use native

"If your application runs [in a supported Cloud Platform](https://docs.spring.io/spring-boot/3.4.2/api/java/org/springframework/boot/cloud/CloudPlatform.html#enum-constant-summary), the server.forward-headers-strategy property defaults to NATIVE. In all other instances, it defaults to NONE."

https://docs.spring.io/spring-boot/how-to/webserver.html